pkg/helm/helmcli.go

```diff
@@ -450,9 +452,11 @@ func GetReleaseValues(kubeInfo *client.KubeInfo, valueKeys []string, releaseName
 		if err != nil {
 			return map[string]interface{}{}, err
 		}
-		for _, valueKey := range valueKeys {
-			if mapVal, ok := valuesMap[valueKey]; ok {
-				values[valueKey] = mapVal
+		if valuesMap != nil {
```

Comment:
This is not related to the orginal issue but I found this could cause nil exception as well.

Reply:
Does it make sense to ensure `GetValuesMap()` returns an empty map instead of nil?

Reply:
Yes - this makes sense, thank you. I have made the changes.